### PR TITLE
Add demo links in footer

### PR DESCRIPTION
**Pull request by owldock**
Per #33  Added links in footer to:
-Payment Request Demo
-Crowdfund App Demo
-Store Demo

first pull request, so be gentle :)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -39,7 +39,7 @@
...
...ITY</h3><ul class="ftb_" id="fBT_-3"><li class="nullTrans"><a class="nullTrans" alt="GitHub" href="https://github.com/btcpayserver/">GitHub</a></li><li><a alt="MATTERMOST" href="http://chat.btcpayserver.org/">Mattermost Chat</a></li><li><a alt="FAQ" href="https://docs.btcpayserver.org/faq-and-common-issues/faq">FAQ</a></li><li><a alt="Troubleshooting" href="https://docs.btcpayserver.org/support-and-community/troubleshooting">Troubleshooting</a></li><li><a alt="Developer docs" href="https://docs.btcpayserver.org/development/">Developer docs</a></li><li><a alt="Community" href="https://docs.btcpayserver.org/support-and-community/community">Community</a></li><li><a alt="Blog" href="https://blog.btcpayserver.org">Blog</a></li><li><a alt="Directory" href="https://directory.btcpayserver.org">Directory</a></li><li><a alt="Support" href="https://docs.btcpayserver.org/support-and-community/support">Support</a></li><li><a alt="Contribute" href="https://docs.btcpayserver.org/support-and-community/contribute">Contribute</a></li><li><a alt="Contribute" href="donate">Donate</a></li></ul></div><p class="copyright_"><a class="nullTrans" alt="Flat 18 Microsystems Development LLC << FLAT18.CO.UK >>"
```
Comment (pavlenex) [inline index.html]: The actual link to the store demo has now been changed to https://store.btcpayserver.org, can you change it? The rest looks good.

Comment (vswee) [reply index.html]: Does that mean that all occurrences of `https://mainnet.demo.btcpayserver.org` including `https://mainnet.demo.btcpayserver.org/``sub-directory/` need updating?

Comment (pavlenex) [reply index.html]: Nope, just store.demo.btcpayserver.org does not work, instead, site has been migrated store.btcpayserver.org

Comment (owldock) [reply index.html]: I couldn't get the Payment Button Demo link to resolve -- neither https://store.demo.btcpayserver.org/index.php/payment-button-demo and https://store.btcpayserver.org/payment-button-demo worked as links. And has the Crowdfund App linked changed too? That one doesn't resolve now either (https://btcpay.kukks.org/apps/pMb1bieuysP3MH2d2s9Yi11Pvb/crowdfund)

Comment (pavlenex) [reply index.html]: @owldock Yes, I think we have to ditch payment button demo until we have  a page for it, will open a separate issue. So just disregard payment button demo, the crowdfunding link works for me though?

Comment (owldock) [reply index.html]: OK cool -- crowdfund link works for me now too, updated the index.html to remove "payment button demo"

**Comment by pavlenex** [comment]
Can you resolve the conflicts?

**Comment by owldock** [comment]
> Can you resolve the conflicts?

I think I did that right, but first time resolving, happy to amend if not. Thanks for your patience! 🙌

**Comment by vswee** [comment]
@owldock I'm hoping you'll be able to replicate this in [r/source/src/html/footer-tmpl.html](https://github.com/btcpayserver/btcpayserver.github.io/blob/master/source/src/html/footer-tmpl.html) so that it can built across all language versions :)

**Pull request merged**